react/features/virtual-background/actions.js

```diff
@@ -37,34 +37,36 @@ export function toggleBackgroundEffect(enabled: boolean) {
 /**
  * Sets the selected virtual background image object.
  *
- * @param {Object} virtualSource - Virtual background image source.
- * @param {boolean} isVirtualBackground - Indicate if virtual image is activated.
+ * @param {Object} options - Represents the virtual background setted options.
  * @returns {{
  *     type: SET_VIRTUAL_BACKGROUND,
  *     virtualSource: string,
  *     isVirtualBackground: boolean,
  * }}
  */
-export function setVirtualBackground(virtualSource: string, isVirtualBackground: boolean) {
+export function setVirtualBackground(options: Object) {
     return {
         type: SET_VIRTUAL_BACKGROUND,
-        virtualSource,
-        isVirtualBackground
+        virtualSource: options.virtualBackground.url,
+        isVirtualBackground: options.virtualBackground.activated
     };
 }
 
 /**
  * Signals the local participant that the background effect has been enabled.
  *
  * @param {boolean} backgroundEffectEnabled - Indicate if virtual background effect is activated.
+ * @param {number} blurValue - Indicate the setted blur value.
  * @returns {{
  *      type: BACKGROUND_ENABLED,
  *      backgroundEffectEnabled: boolean,
+ *      blurValue: number
  * }}
  */
-export function backgroundEnabled(backgroundEffectEnabled: boolean) {
+export function backgroundEnabled(backgroundEffectEnabled: boolean, blurValue: number) {
```

Comment:
The blur value should be part of the virtual background options, this action should only control if the virtual background is going to be enabled or not.